src/Symfony/Component/Console/SignalRegistry/SignalRegistry.php

```diff
@@ -49,4 +49,20 @@ public function handle(int $signal): void
             $signalHandler($signal, $hasNext);
         }
     }
+
+    /**
+     * @internal
```

Comment:
Do we need to make this function internal? Checking if signal handling is generally available is probably something userland code might want to do as well.

Reply:
Works for me :) flag removed